openhands/runtime/utils/wrappers/oh-chromium

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# /openhands/bin/oh-chromium
```

Comment:
I'm not seeing oh_chromium called anywhere, isn't it just run directly by playwright?

Reply:
Yeah - you can probably ignore the detailed implementation now :)
Right now this PR should just be a rough idea / design doc, where we might use different things for actual implementation.

Maybe we could using things like this: https://pypi.org/project/poetry-pyinstaller-plugin/